raft-ann-bench/cpu/Dockerfile

```
@@ -9,26 +9,32 @@ ARG PYTHON_VER
 
 COPY condarc /opt/conda/.condarc
 
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
 # Create a data folder accessible by any user so mounted volumes under it can be accessible
 # when the user passes their uid to docker run with -u $(id -u)
 # Also add the conda_prefix config to the global bashrc file so that all users have it correctly configured.
-RUN mkdir /data && chmod 777 /data \
-    &&  ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh; \
-    echo ". /opt/conda/etc/profile.d/conda.sh; conda activate base" >> /etc/bash.bashrc
+RUN <<EOF
+mkdir /data
+chmod 777 /data
+echo ". /opt/conda/etc/profile.d/conda.sh; conda activate base" >> /etc/bash.bashrc
+EOF
 
 # we need perl temporarily for the remaining benchmark perl scripts
 RUN apt-get install perl -y
 
 # Install python before updating environment, otherwise Python 3.9 image
 # runs into a solver conflict with truststore 0.8.0. This avoids the environment installing
 # packages incompatible with python version needed before python itself is pinned to the correct version.
-RUN mamba install -y -n base "python=${PYTHON_VER}" \
-    && mamba update --all -y -n base \
-    && mamba install -y -n base \
-    "raft-ann-bench-cpu=${RAPIDS_VER}.*" \
-    "python=${PYTHON_VER}" \
-    && conda clean -afy \
-    && chmod -R 777 /opt/conda
+RUN <<EOF
+mamba install -y -n base "python=${PYTHON_VER}"
+mamba update --all -y -n base
+mamba install -y -n base
+"raft-ann-bench-cpu=${RAPIDS_VER}.*" \
+"python=${PYTHON_VER}"
```

```suggestion
mamba install -y -n base \
  "raft-ann-bench-cpu=${RAPIDS_VER}.*" \
  "python=${PYTHON_VER}"
```

Looks like a valid `\` was inadvertently removed.